DOCS-3288: import partner and community integrations (#2188)
MultiSafepay/docs: content/integrations/community-integrations.md

[View in store](https://www.nopcommerce.com/en/multisafepay-payment-plugin).

___

# Omnipay
<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/Omnipay.svg" width="50" align ="right" style="transform: translate(-50%, -50%);"/>

The League of Extraordinary Packages provides a MultiSafepay driver for the Omnipay PHP payment processing library.

[View on GitHub](https://github.com/thephpleague/omnipay-multisafepay).

___

# OroCommerce

<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/OroCommerce.svg" width="50" align ="right" style="transform: translate(-50%, -50%);"/>

Casper provides a MultiSafepay extension for OroCommerce.

[View in store](https://extensions.oroinc.com/orocommerce/extension/multisafepay-bundle/#description).

___

# Sylius

<img src="https://raw.githubusercontent.com/MultiSafepay/docs/master/static/logo/Integrations/Sylius.svg.svg" width="50" align ="right" style="transform: translate(-50%, -50%);"/>

BitBag provides a MultiSafepay plugin for Sylius.

[View on GitHub](https://github.com/BitBagCommerce/SyliusMultiSafepayPlugin).

___

# Build a community integration

> At MultiSafepay, our developers write code to process in-house transactions. However, there are also many fantastic developers worldwide who create code wrappers, plugins, and applications in order to integrate our products and services.

Reviewer: ```suggestion
At MultiSafepay, our developers write code  in-house for processing transactions. However, there are many fantastic external developers worldwide who create code wrappers, plugins, and applications to integrate our products and services.
```
